Formula/sheldon.rb

```diff
@@ -28,6 +28,14 @@ class Sheldon < Formula
   end
 
   def install
+    # Ensure the declared `openssl@1.1` dependency will be picked up.
+    # https://docs.rs/openssl/latest/openssl/#manual
+    ENV["OPENSSL_DIR"] = Formula["openssl@1.1"].opt_prefix
+    ENV["OPENSSL_NO_VENDOR"] = "1"
+
+    # Replace vendored `libgit2` with our formula
+    inreplace "Cargo.toml", /features = \["vendored-libgit2"\]/, "features = []"
```

Comment:
You don't really need a regex here.
```suggestion
    inreplace "Cargo.toml", 'features = ["vendored-libgit2"]', "features = []"
```